app/scripts/lib/multichain-api/scope/scope.ts

```diff
@@ -49,18 +49,32 @@ export const KnownNotifications: Record<NonWalletKnownCaipNamespace, string[]> =
     eip155: ['accountsChanged', 'chainChanged', 'eth_subscription'],
   };
 
-export type Scope = CaipChainId | CaipReference;
+// These External prefixed types represent the CAIP-217
+// Scope and ScopeObject as defined in the spec.
+export type ExternalScope = CaipChainId | CaipReference;
```

Comment:
Ok sorry to switch up again on ya here but wdyt about `ExternalScopeString` and `ScopeString` that's actually [what they're called in CAIP-217](https://github.com/ChainAgnostic/CAIPs/blob/6cf6408c50865b596258f2de476133670aa9aea7/CAIPs/caip-217.md?plain=1#L54) and CAIP-25